packages/block-editor/src/components/block-draggable/style.scss

```diff
@@ -45,8 +45,18 @@
 	}
 }
 
-// This hides the block being dragged.
-// The effect is that the block being dragged appears to be "lifted".
+// This creates a "slot" where the block you're dragging appeared.
+// We use !important as one of the rules are meant to be overriden.
 .is-dragging {
```

Comment:
That look like a very generic selector, it could affect a lot of things (including things not rendered by core Gutenberg), should we make it more specific?

Reply:
Good point. Let me see if I can scope it better, I know that would make @gwwar happy as well 😅

Reply:
Looks like I can scope it like so: `.block-editor-block-list__layout .is-dragging` — would that be preferable? It doesn't feel like the right file to store that particular rule for. 

The `!important`s still feel correct to me insofar as the way we make the "slot" appear is that we style and override the appearance of the block itself.

Reply:
For me `.block-editor-block-list__block.is-dragging` sounds like a better selector. Also, it seems `.is-dragging` class is used in the ListView as well, so we might want to check whether there's any regression there by these changes.